### ISSUE TITLE
Require image clipping

### ISSUE THREAD

**Issue by patryk-sredzinski** [longitachi/ZLPhotoBrowser#838]
### Feature Description
I am in a need to force user to crop image to square/circle (when he edits a profile avatar)
I don't think I have seen this feature available. Planning to modify it.

Can I get any suggestions so in the end it get's merged?

**Comment by longitachi** [comment]
```swift
ZLPhotoConfiguration.default()
    .editImageConfiguration
    .clipRatios([.circle]) // or  .clipRatios([.wh1x1])
```

**Comment by patryk-sredzinski** [comment]
This I believe only adds clip ratio, but user is still able to not to enter clipping controller, and keep original photo.

I'd like to make him forced to clip, and also allow editing the image

**Comment by longitachi** [comment]
OK, this configuration can meet your needs
```Swift
ZLPhotoConfiguration.default()
    .editImageConfiguration
    .tools([.clip])
    .clipRatios([.circle]) // or  .clipRatios([.wh1x1])

ZLPhotoConfiguration.default()
    .maxSelectCount(1)
    .editAfterSelectThumbnailImage(true)
    .showClipDirectlyIfOnlyHasClipTool(true)
```

**Comment by patryk-sredzinski** [comment]
Yeah, but I believe this will not let me do other editing on image right? No Drawings, no sticker, no filters.

That's why I think I'd like to add this functionality, but I'd like you to help me planning it so it works well.

**Comment by longitachi** [comment]
Delete `.tools([.clip])`

**Comment by patryk-sredzinski** [comment]
Yeah, that's why I'd like to ask you what do you think I can do as a new functionality for it to be supported properly.
